packages/babel-plugin/src/css-prop/index.tsx

```diff
@@ -26,20 +52,25 @@ const getJsxAttributeExpression = (node: t.JSXAttribute) => {
  */
 export const visitCssPropPath = (path: NodePath<t.JSXOpeningElement>, meta: Metadata): void => {
   let cssPropIndex = -1;
-  const cssProp = path.node.attributes.find((attr, index): attr is t.JSXAttribute => {
-    if (t.isJSXAttribute(attr) && attr.name.name === 'css') {
+  const cssProp = path.get('attributes').find((attr, index): attr is NodePath<t.JSXAttribute> => {
```

Comment:
Oh nice, does `.get` allow you to traverse the `NodePath` Object to find a property?

Reply:
yeah, I think this is something we're doing wrong throughout the repo. Once you unbox the path into a node, there's no way to get it back, so we should probably pass around paths instead.